ConsoleHandler.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.handler;
+
+public interface ConsoleHandler {
+
+    static void handle(){ }

```

Comment:
empty static method in interface , why ?

Reply:
> empty static method in interface , why ?

I needed a common interface, a static method must have a body, because I declared it empty.

Reply:
why it is static? Do you know that static methods are not inherited ?